Makefile

```diff
@@ -14,7 +14,7 @@ endif
 ifneq ("$(wildcard /usr/share/selinux/devel/include/*/docker.if)","")
 export M4PARAM += -D has_docker
 else ifneq ("$(wildcard /usr/share/selinux/devel/include/*/container.if)","")
-export M4PARAM += -D has_docker
+export M4PARAM += -D has_container
 else ifneq ($(DISTRO),rhel6)
 $(error *** Interface container.if or docker.if not present, cannot continue ***)
```

Comment:
Flags has_docker or has_container are unused but we still error out when these are not present.